assets/js/nav-scroll.js

```diff
@@ -1,12 +1,27 @@
 let siteNav = document.querySelector('.site-nav');
+const key = 'scroll';
 
 document.addEventListener('DOMContentLoaded', () => {
-    const scroll = localStorage.getItem('scroll');
-    if (scroll !== null) {
-        siteNav.scrollTop = parseInt(scroll);
+    const scroll = JSON.parse(localStorage.getItem(key));
+
+    const currentDate = new Date();
+    
+    if (scroll !== null && currentDate.getTime() < scroll.expiry) {
+      siteNav.scrollTop = parseInt(scroll.value);
+    }
+    else {
+      localStorage.removeItem(key);
     }
 });
 
 window.addEventListener('beforeunload', () => {
-  localStorage.setItem('scroll', siteNav.scrollTop);
-});
+  const currentDate = new Date();
+
+  // add the scroll value that expires after one day
+  const scroll = {
+    value: siteNav.scrollTop,
+    expiry: currentDate.getTime() + 24 * 60 * 60 * 1000,
+  }
+  
+  localStorage.setItem(key, JSON.stringify(scroll));
```

Comment:
Do we really want this across sessions? IMO it is more appropriate to use `sessionStorage` for this.

Reply:
Changed to sessionStorage. Thank you!